torch/storage.py

```diff
@@ -192,18 +192,23 @@ def complex_float(self):
         return self._to(torch.cfloat)
 
     def pin_memory(self, device="cuda"):
```

Comment:
We should remove the `def is_pinned(self) -> bool: ...  # noqa: E704` line above. Also might as well add the `-> bool`  to this line